src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/TraceLogging/NameInfo.cs

```diff
@@ -82,14 +82,14 @@ private int Compare(string otherName, EventTags otherTags)
         }
 
 #if FEATURE_PERFTRACING
-        public IntPtr GetOrCreateEventHandle(EventProvider provider, ConcurrentDictionary<int, IntPtr> eventHandleMap, EventDescriptor descriptor, TraceLoggingEventTypes eventTypes)
+        public IntPtr GetOrCreateEventHandle(EventProvider provider, TraceLoggingEventHandleTable eventHandleTable, EventDescriptor descriptor, TraceLoggingEventTypes eventTypes)
         {
             IntPtr eventHandle = IntPtr.Zero;
```

Comment:
Nit: doesn't need to be initialized to IntPtr.Zero, as it's assigned on the next line

Reply:
Fixed.